### PR TITLE
Alisaie addition

### PR DESCRIPTION
**Pull request by Automata02**
## Description

Adding Alisaie keyboard to the QMK repository.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
None.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/automata02/alisaie/config.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
```
Comment (elpekenin) [inline keyboards/automata02/alisaie/config.h:22]: These values get inferred from the pins defined on the JSON, thus this file contains 0 info and shall be deleted.

File: keyboards/automata02/alisaie/info.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
+    "manufacturer": "Automata02",
+    "keyboard_name": "Alisaie",
+    "maintainer": "qmk",
+    "bootloader": "atmel-dfu",
+    "bootmagic": {
+        "enabled": true
+    },
+    "debounce": 5,
```
Comment (elpekenin) [inline keyboards/automata02/alisaie/info.json:9]: Default value, dont explicitly add it (useless code)

File: keyboards/automata02/alisaie/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        	_______, _______,                   _______, _______, _______, _______, _______,                            _______, _______
+    ),
+    [3] = LAYOUT_split_bs_rshift(
+        	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                   _______,
+        	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        	_______, _______,                   _______, _______, _______, _______, _______,                            _______, _______
+    )
```
Comment (elpekenin) [inline keyboards/automata02/alisaie/keymaps/default/keymap.c:48]: Empty, read: useless, layers should be removed. They aren't even reachable

File: keyboards/automata02/alisaie/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN,          KC_PSCR, _______,
+        	_______, _______,                   _______, _______, _______, _______, _______,                            _______, _______
+    ),
+    [3] = LAYOUT_split_bs_rshift(
+        	QK_BOOT,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,
+        	_______, _______, KC_PGUP, _______, _______, _______, _______, _______, KC_UP,   _______, KC_MPRV, KC_UP,   KC_MNXT,          _______,
+        	_______, KC_HOME, KC_PGDN, KC_END,  _______, KC_VOLD, KC_VOLU, KC_LEFT, KC_DOWN, KC_RGHT, KC_LEFT, KC_RGHT,                   _______,
+        	_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN,          KC_PSCR, _______,
+        	_______, _______,                   _______, _______, _______, _______, _______,                            _______, _______
+    ),
```
Comment (elpekenin) [inline keyboards/automata02/alisaie/keymaps/via/keymap.c:48]: Same as before, you cant event get here and they are just copies of layer `1`

File: keyboards/automata02/alisaie/readme.md [hunk 5]
```diff
@@ -0,0 +1 @@
+60% Alice style keyboard.
```
Comment (elpekenin) [inline keyboards/automata02/alisaie/readme.md:1]: This file should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/docs/documentation_templates.md)

**Review by elpekenin — CHANGES_REQUESTED** [review]

**Comment by Automata02** [comment]
@elpekenin readme has been updated and redundant entries removed.

**Review by drashna — APPROVED** [review]

File: keyboards/automata02/alisaie/info.json [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+        "nkro": true,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "cols": ["B6", "F0", "F1", "F4", "F5", "B5", "B7", "D3", "D2", "D1", "D0", "B3", "B2", "B0"],
+        "rows": ["E6", "F6", "D5", "D4", "B1"]
+    },
+    "mouse_key": {
+        "enabled": false
+    },
```
Comment (waffle87) [inline keyboards/automata02/alisaie/info.json:27]:
```suggestion
    "diode_direction": "COL2ROW",
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "nkro": true,
    },
    "matrix_pins": {
        "cols": ["B6", "F0", "F1", "F4", "F5", "B5", "B7", "D3", "D2", "D1", "D0", "B3", "B2", "B0"],
        "rows": ["E6", "F6", "D5", "D4", "B1"]
    },
```

File: keyboards/automata02/alisaie/readme.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+![Alisaie layout](https://i.imgur.com/0YsrKvw.png)
+
+A 60% alice like keyboard sharing the same footprint and 8 degree alpha rotation as Meridian but with a tsangan bottom row.
+
+* Keyboard Maintainer: [Automata02](https://github.com/Automata02/)
+* Hardware Supported: Alisaie PCB version 0.0.4b
+
+Make example for this keyboard (after setting up your build environment):
+
+> make automata02/alisaie:default
```
Comment (waffle87) [inline keyboards/automata02/alisaie/readme.md:12]:
```suggestion
Make example for this keyboard (after setting up your build environment):

    make automata02/alisaie:default

Flashing example for this keyboard:

    make automata02/alisaie:default:flash
```

File: keyboards/automata02/alisaie/rules.mk
Comment (waffle87) [inline keyboards/automata02/alisaie/rules.mk:1]: This file should match [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/rules.mk).

File: keyboards/automata02/alisaie/info.json [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x2012",
+        "vid": "0xBABE"
+    },
+    "layouts": {
+        "LAYOUT_split_bs_rshift": {
```
Comment (waffle87) [inline keyboards/automata02/alisaie/info.json:35]: This change will also need to be reflected in the keymaps.
```suggestion
        "LAYOUT": {
```

Comment (Automata02) [reply keyboards/automata02/alisaie/info.json]: Has been updated. Physically board supports split and non split so I would have liked to kept it but I can revert at a later date when and if I make a second layout.

**Pull request merged**